feat: реализация приложения "Заметки"

timachoo opened the pull request:
Реализация приложения "Заметки"

timachoo committed:
feat: реализация приложения "Заметки"

gusar-off commented on src/main/kotlin/CreareNote.kt:
```diff
@@ -0,0 +1,21 @@
+            archive.addNote(noteName!!, noteText!!)
+            exit()
+        } else {
+            println("Вы не ввели текст, попробуйте еще")
```
Повторяется выше, лучше вынести в метод

gusar-off commented on src/main/kotlin/CreateArchiveScreen.kt:
```diff
@@ -0,0 +1,15 @@
+            MenuNavigation.archives.add(Archive(archName!!))
+            exit()
+        } else {
+            println("Вы не ввели текст, попробуйте еще")
```
И тут

gusar-off commented on src/main/kotlin/Screens.kt:
```diff
@@ -0,0 +1,8 @@
+enum class Screens {
+    ARCHIVE
+    , CREATE_ARCHIVE
```
Лучше запятую не переносить, читается лучше

timachoo committed:
fix:
1. в Screens.kt исправлено форматирование
2. Печать ошибки при пустом вводе вынесена в функцию
